src/System.Net.Http/src/System/Net/Http/CurlHandler/CurlHandler.MultiAgent.cs

```diff
@@ -999,6 +999,12 @@ private static ulong CurlReceiveHeadersCallback(IntPtr buffer, ulong size, ulong
                                     // our redirect limit, etc.), this will have been unnecessary work in reconfiguring the easy handle, but 
                                     // nothing incorrect, as we'll tear down the handle once the request finishes, anyway, and all of the configuration
                                     // we're doing is about initiating a new request.
+                                    if ((int)response.StatusCode >= 301 && (int)response.StatusCode <= 303)
+                                    {
+                                        // ISSUE: 25163
+                                        // Ideally we want to avoid modifying the users request message.
+                                        easy._requestMessage.Headers.TransferEncodingChunked = false;
```

Comment:
Is `false` what we want, or would `null` be better?

Reply:
I think `false` is the right choice, based off the code that blocks the header:
https://github.com/dotnet/corefx/blob/52bba5248ba4105140f8f62359da7779d8860d0b/src/System.Net.Http/src/System/Net/Http/CurlHandler/CurlHandler.EasyRequest.cs#L732-L738
It's also what `CurlHandler` does alsewhere:
https://github.com/dotnet/corefx/blob/444e6bb7fd274be680fb3bf4b22258821e77ae2c/src/System.Net.Http/src/System/Net/Http/CurlHandler/CurlHandler.cs#L777-L781